app/models/agents/website_agent.rb

```diff
@@ -439,7 +441,14 @@ def extract_xml(doc)
         case nodes
         when Nokogiri::XML::NodeSet
           result = nodes.map { |node|
-            case value = node.xpath(extraction_details['value'] || '.')
+            value = node.xpath(extraction_details['value'] || '.')
+            if extraction_details['strip_cdata']
```

Comment:
I'm pretty tempted to say that we should make this whole block into the following: 

```ruby
value = node.xpath(extraction_details['value'] || '.')
value = value.first.text if value.first && value.first.cdata?
value = value.to_i if value.is_a?(Float) && value.to_i == value
value.to_s
```

this cleans up the single-case `case` statement and also always does cdata unwrapping.